### PR TITLE
cloud: allow to select identity certificate chain id

### PR DESCRIPTION
**Pull request by jkralik**

### CODE REVIEW

File: include/oc_cloud.h [hunk 1]
```diff
@@ -213,6 +215,24 @@ int oc_cloud_provision_conf_resource(oc_cloud_context_t *ctx,
...
                                      const char *auth_provider);
+/**
+ * @brief Set identity certificate chain for establish TLS connection.
+ *
+ * @param ctx Cloud context to update, must not be NULL.
+ * @param selected_identity_cred_id Selected identity certificate chain id.
+ * -1(default) means any.
+ */
+
+void oc_cloud_set_identity_cert_chain(oc_cloud_context_t *ctx,
```
Comment (Danielius1922) [inline include/oc_cloud.h:226]: OC_API missing, I think

File: include/oc_cloud.h [hunk 2]
```diff
@@ -213,6 +215,24 @@ int oc_cloud_provision_conf_resource(oc_cloud_context_t *ctx,
...
+                                      int selected_identity_cred_id);
+/**
+ * @brief Get selected identity certificate chain for establish TLS connection.
```
Comment (Danielius1922) [inline include/oc_cloud.h:229]: chain **for** establish -> chain **to** establish

File: include/oc_cloud.h (hunk 1 above)
Comment (jkralik) [reply include/oc_cloud.h:226]: fixed

File: include/oc_cloud.h [hunk 3]
```diff
@@ -213,6 +215,24 @@ int oc_cloud_provision_conf_resource(oc_cloud_context_t *ctx,
                                      const char *access_token,
                                      const char *server_id,
                                      const char *auth_provider);
+/**
+ * @brief Set identity certificate chain for establish TLS connection.
```
[the remark this answers is not in the document]
Comment (jkralik) [reply include/oc_cloud.h]: fixed

**Pull request merged**